src/site/content/en/fast/lazy-loading-images/index.md

```diff
@@ -0,0 +1,302 @@
+  - jeremywagner
+  - rachelandrew
+date: 2019-08-16
+updated: 2020-06-09
+description: |
+  This post explains lazy-loading and the options available to you when lazy-loading images and video.
+tags:
+  - performance
+---
+
+Images can appear on a webpage due to being inline in the HTML as `<img>` elements
+or as CSS background images. In this guide you will find out how to lazy-load both types of image.
+
+### Inline images {: #images-inline }
+
+The most common lazy-loading candidates are images as used in `<img>` elements.
+With inline images we have three options for lazy-loading,
+which may be used in combination for the best compatibility across browsers.
+
+- [Using native browser lazy-loading](#images-inline-native).
+- [Using intersection observer](#images-inline-intersection-observer).
+- [Using scroll and resize event handlers](#images-inline-event-handlers).
+
+#### Using native browser lazy-loading {: #images-inline-native }
+
+Chrome and Firefox both support native browser lazy-loading with the `loading` attribute.
+This attribute can be added to `<img>` elements, and also to `<iframe>` elements.
+A value of `lazy` tells the browser to load the image immediately if it is in the viewport,
+and to fetch other images when the user scrolls near them.
+
+{% Aside %}
+  Please note `<iframe loading=lazy>` is currently non-standard.
+  While implemented in Chromium, it does not yet have a specification and is subject to future change when this does happen.
+  We suggest not to lazy-load iframes using the loading attribute until it becomes part of the specification.
+{% endAside %}
+
+See the `loading` field of MDN's
+[Browser compatibility](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Browser_compatibility)
+table for details of browser support for native lazy-loading.
+If the browser does not support native lazy-loading then the attribute will be ignored
+and images will load immediately, as normal.
+
+For most websites, adding this attribute to inline images will be a performance boost
+and save users loading images that they may not ever scroll to.
+If you have large numbers of images and want to be sure that users of browsers without support for native lazy-loading benefit
+you will need to combine this with one of the methods explained next.
+
+#### Using intersection observer {: #images-inline-intersection-observer }
+
+To polyfill lazy-loading of `<img>` elements, we use JavaScript to check if they're in the
+viewport. If they are, their `src` (and sometimes `srcset`) attributes are
+populated with URLs to the desired image content.
+
+If you've written lazy-loading code before, you may have accomplished your task
+by using event handlers such as `scroll` or `resize`. While this approach is the
+most compatible across browsers, modern browsers offer a more performant and
+efficient way to do the work of checking element visibility via [the
+intersection observer API](/web/updates/2016/04/intersectionobserver).
```

Comment:
```suggestion
intersection observer API](https://developers.google.com/web/updates/2016/04/intersectionobserver).
```

Reply:
Double-check that all the links are working, I've also noticed a link that's broken because it used to be an anchor link on Web Fundamentals version.